http/test.http

```diff
@@ -2,10 +2,13 @@ GET http://localhost:7071/api/httpget HTTP/1.1
 
 ###
 
+GET http://localhost:7071/api/httpget?name=Developer HTTP/1.1
+
+###
+
 POST http://localhost:7071/api/httppost HTTP/1.1
 content-type: application/json
 
 {
-    "name": "Awesome Developer",
-    "age": 25 
```

Comment:
Same as above, liked the name and age Person object.  not sure why changing.  then the rest need to change too and not sure if better.